### ISSUE TITLE
Update US emissions to use 2021 data? 

### ISSUE THREAD

**Issue by helicopter-z** [electricitymaps/electricitymaps-contrib#5716]
## Description
Hi, I see on https://github.com/electricitymaps/electricitymaps-contrib/wiki/US-emission-factors that we are currently using 2020 data. 2021 data is available https://www.epa.gov/egrid/download-data. Should I go ahead and update it?

**Comment by VIKTORVAV99** [comment]
There is already a open issue about this at https://github.com/electricitymaps/electricitymaps-contrib/issues/5417 so closing this as a duplicate in favor of the original issue.

**Comment by VIKTORVAV99** [comment]
Hello, sorry for closing this and reopening it. I thought we already had a issue for this but apparently  that was for the EU 😅

I have created a new tracking issue that links to this so it will be easier to keep track of in the future.

I don't think anyone is working on it right now but @w-flo has expressed some interest in it so perhaps you guys could coordinate some?

I also don't think it is as straight forward as it seems. Some generation only zones have been merged with their parent zones and should be re-mapped to the parent zone in the scripts so we get the right emission factor (this should not have a big impact but let's strive for accuracy).
